src/Vanilla/Cli/Command/BuildCmd.php

```diff
@@ -211,14 +211,17 @@ protected function setBuildOptionsFromArgs(Args $args) {
      * @return array
      */
     protected function getEnabledAddonKeys() {
-        $filePath = $this->vanillaSrcDir.'/conf/' . $this->configName;
-        if (!file_exists($filePath)) {
-            CliUtil::fail("Unable to locate Vanilla configuration at $filePath.");
+        $configPath = $this->vanillaSrcDir . '/conf/' . $this->configName;
```

Comment:
We are using no space when concatening

`$this->vanillaSrcDir.'/conf/'.$this->configName`